### PR TITLE
6.4 version fix and mergeback 6.3 hotfixes

### PR DESCRIPTION
**Pull request by stanleytsang-amd**

### CODE REVIEW

File: CHANGELOG.md
Comment (NguyenNhuDi) [inline CHANGELOG.md:8]: It seems my changelog update for extended tests is missing.
```suggestion
* Added extended tests to `rtest.py`. These tests are extra tests that did not fit the criteria of smoke and regression tests. These tests will take much longer to run relative to smoke and regression tests. Use `python rtest.py [--emulation|-e|--test|-t]=extended` to run these tests.
* Added regression tests to `rtest.py`. These tests recreate scenarios that have caused hardware problems in past emulation environments. Use `python rtest.py [--emulation|-e|--test|-t]=regression` to run these tests.
```

**Review by NguyenNhuDi — CHANGES_REQUESTED** [review]

**Review by RobsonRLemos — APPROVED** [review]
It looks good for 6.4 version fix.

**Review by NguyenNhuDi — APPROVED** [review]
Should be good after you add back the extended tests in the changelog update. Approving now so that you don't have to wait for my approval after you make the changes.

File: CHANGELOG.md
Comment (stanleytsang-amd) [reply CHANGELOG.md:8]: Whoops..Added the extended test entry.

**Pull request merged**
